decidim-verifications/app/events/decidim/verifications/managed_user_error_event.rb

```diff
@@ -26,7 +25,13 @@ def resource_title
       end
 
       def default_i18n_options
-        super.merge({ conflicts_path: conflicts_path, managed_user_path: managed_user.profile_path, managed_user_name: managed_user.name })
+        super.merge({
+                      conflicts_path: conflicts_path,
+                      conflicts_url: conflicts_url,
+                      managed_user_path: managed_user.profile_path,
+                      managed_user_url: managed_user.profile_url,
```

Comment:
The participant related URLs are still pointing to `http://localhost` without the port configuration. This also applies to the methods delegated to `updated_user`.

I think we also need to apply the same fix here as well to fix those:
https://github.com/decidim/decidim/blob/c1dd072503ddd4f830073d8c7038de30a70a7306/decidim-core/app/presenters/decidim/user_presenter.rb#L28

Noticed this when looking at the spec which still expected these URLs without the port, so it needs to be updated there as well. Or if you want, we can handle that in another PR since it may also cause changes elsewhere?

Reply:
At the same time, I also noticed that the methods here are also calling the `_path` helpers which means that these default URLs would probably not work in actual production emails:
https://github.com/decidim/decidim/blob/c1dd072503ddd4f830073d8c7038de30a70a7306/decidim-core/app/uploaders/decidim/avatar_uploader.rb#L15

This is definitely not related to this PR but I just wanted to note it here as I noticed it at the same time.